test/e2e/reboot.go

```diff
@@ -126,8 +115,7 @@ func rebootNode(c *client.Client, provider, name string, result chan bool) {
 	}
 
 	// Get all the pods on the node.
-	podList, err := c.Pods(api.NamespaceDefault).List(
-		labels.Everything(), fields.OneTermEqualSelector(client.PodHost, name))
+	podList, err := listPods(c, api.NamespaceDefault, labels.Everything(), fields.OneTermEqualSelector(client.PodHost, name))
```

Comment:
Agh, I should have looked into what this did before switching to it.

We don't want to create our own namespace for this test. This test is supposed to look at all of the infrastructure pods (in the default namespace) and ensure that they are running both before and after the cluster is restarted. While other tests should be isolated by namespace, this test is lower level than most tests,.